app/coffee/Features/ThirdPartyDataStore/TpdsUpdateHandler.coffee

```diff
@@ -24,10 +26,15 @@ module.exports =
 					cb err, project
 		getOrCreateProject (err, project)->
```

Comment:
This call desearlises the project, negating the performance benefit. Need to kill the request before this point.